gen/calc.py

```diff
@@ -339,6 +339,26 @@ def validate_bootstrap_tmp_dir(bootstrap_tmp_dir):
         "Must be an absolute path to a directory, although leave off the `/` at the beginning and end."
 
 
+def calculate_minuteman_min_named_ip_erltuple(minuteman_min_named_ip):
```

Comment:
Yeah, I also didn't like it but then my previous PR is on the master itself (didn't create a branch for that change) and this one is on the branch cut out of the master after the previous commit. Is it fine if we just let it be this way this time? From next time, I plan to create branches for individual changes to avoid such issues.